docs/upgrade-to-v5.md

```diff
@@ -179,3 +179,33 @@ EJSON.parse("...",  { strict: false }); /* migrate to */ EJSON.parse("...",  { r
 // stringify
 EJSON.stringify({}, { strict: true  }); /* migrate to */ EJSON.stringify({}, { relaxed: false });
 EJSON.stringify({}, { strict: false }); /* migrate to */ EJSON.stringify({}, { relaxed: true });
+
+### `Code` only stores a string
```

Comment:
It doesn’t look like support for `Code` with `scope` has gone away here, so this might be a confusing wording?

Reply:
good call, fixed up the wording here